movethebytes/data-transfer/krakensync/_compositions/krakensync.toml

```diff
@@ -0,0 +1,19 @@
+[metadata]
+  name = "krakensync"
+
+[global]
+  plan = "krakensync"
+  case = "one_to_one"
+  builder = "docker:generic"
+  runner = "local:docker"
+  total_instances = 2
+  disable_metrics = false
+
+[[groups]]
+  id = "group1"
+  [groups.instances]
+    count = 2
+  [groups.run]
+    [groups.run.test_params]
+      latency = "100"
+      bandwidth = "10485760"
```

Comment:
This is 10 Mbps, we ran so far with 1Gbps not to have network be the constraint. We will do more network based stressing at some later stages.
Same for latency since it's mildly broken, we just use 10 to keep the oscillations on latency small.

Reply:
Okay, I took these #s from https://github.com/n0-computer/test-plans/blob/21d4f9b08adc0c34478bcd130209df37cc0b79a5/movethebytes/data-transfer/_compositions/bitswap.toml#L18-L20, are those not the correct numbers then?

Reply:
`10ms` & `1GiB` is what I used for the MTBWG Reports.